pre/remap_restart/remap_command_line.py

```diff
@@ -81,7 +81,12 @@ def parse_args(program_description):
     p_command.add_argument('-qos',        default="",    help="slurm_pbs quality-of-service", choices=['', 'debug', 'allnccs', 'normal'])
     account = get_account()
     p_command.add_argument('-account',    default=account,    help="slurm_pbs account")
-    p_command.add_argument('-partition',  default='',       help="slurm_pbs partition")
+
+    if (BUILT_ON_SLES15):
```

Comment:
@biljanaorescanin I might have removed these. These references to `scutest` should no longer be needed now that SCU17 is in general availability.

So just having a "blank" partition default is fine.

Reply:
I think I removed all scutest with last commit.